test/1_stdlib/Unmanaged.swift

```diff
@@ -54,6 +54,29 @@ UnmanagedTests.test("_withUnsafeGuaranteedRef/return") {
   }
 }
 
+UnmanagedTests.test("toOpaque") {
+  var ref = Foobar()
+  let unknownPtr = unsafeBitCast(
+    Unmanaged.passUnretained(ref).toOpaque(),
+    to: Int.self) 
```

Comment:
Isn't this `let unknownPtr = Int(bitPattern: Unmanaged.passUnretained(ref).toOpaque())`?  We prefer to avoid `unsafeBitCast()`.